orchestra/static/orchestra/todos/todo-list.html

```diff
@@ -3,83 +3,85 @@
     <div class="row section-header">
       <div class="col-lg-12 col-md-12 col-sm-12">
         <h3>
-          Todo List
-          <a class="btn"
-               ng-if="todoList.canSendToPending()"
-               ng-click="todoList.sendToPending()">
-            Send to pending
-          </a>
+          Checklist
         </h3>
       </div>
     </div>
     <div class="row section-body" ng-if="todoList.ready">
-      <div class="col-lg-12 col-md-12 col-sm-12">
-        <form class="new-todo">
-          <div class="new-todo__box">
+      <div class="col-lg-6 col-md-6 col-sm-12">
+        <div class="checklist-pane">
```

Comment:
since everything in Orchestra is called todos, let's keep naming to todo/todolist

Reply:
ok :)